core/egov-localization/src/main/java/org/egov/web/controller/MessageController.java

```diff
@@ -79,9 +81,35 @@ private MessagesResponse createResponse(List<org.egov.domain.model.Message> doma
 		return getSuccessResponse(messages);
 	}
     
+    @PostMapping(value = "/_update")
+	@ResponseBody
+	public ResponseEntity<?> update(@RequestBody @Valid final NewMessagesRequest messagesRequest,
+			final BindingResult bindingResult) {
+		if (bindingResult.hasErrors()) {
+			throw new InvalidCreateMessageRequest(bindingResult.getFieldErrors());
+		}
+		logger.info("Update Message Request:" + messagesRequest);
+		messageService.createMessage(messagesRequest);
+		final List<Message> messages = messagesRequest.getMessages();
+		return getSuccessResponse(messages);
+	}
+    
+    @PostMapping(value = "/_delete")
+	@ResponseBody
+	public ResponseEntity<?> delete(@RequestBody @Valid final DeleteMessagesRequest deleteMessagesRequest,
+			final BindingResult bindingResult) {
+		if (bindingResult.hasErrors()) {
+			throw new InvalidCreateMessageRequest(bindingResult.getFieldErrors());
+		}
+		logger.info("Delete Message Request:" + deleteMessagesRequest);
+		messageService.deleteMessage(deleteMessagesRequest);
+		final List<Message> messages = deleteMessagesRequest.getMessages();
+		return getSuccessResponse(messages);
+	}
+    
     private ResponseEntity<?> getSuccessResponse(final List<Message> messages) {
 		final MessagesResponse messageResponse = new MessagesResponse(messages); 
-		return new ResponseEntity<>(messageResponse, HttpStatus.OK);
+		return new ResponseEntity<>(messageResponse, HttpStatus.CREATED);
```

Comment:
@darshannagesh Since we are not being strictly Restful the only 2 codes we are using are 200 and 400.